src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EEMethodSymbol.cs

```diff
@@ -711,13 +711,7 @@ internal override void AddSynthesizedReturnTypeAttributes(PEModuleBuilder module
 
             if (this.ReturnType.ContainsNullableReferenceTypes())
             {
-                var diagnostics = DiagnosticBag.GetInstance();
-                SourceAssemblySymbol.GetUseSiteDiagnosticsForNullableAttribute(compilation, diagnostics);
-                if (!diagnostics.HasAnyErrors())
-                {
-                    AddSynthesizedAttribute(ref attributes, compilation.SynthesizeNullableAttribute(this.ReturnType));
-                }
-                diagnostics.Free();
+                AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeNullableAttribute(this, this.ReturnType));
```

Comment:
>AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeNullableAttribute(this, this.ReturnType)); [](start = 16, length = 106)

It looks like this line is going to add a duplicate attribute, the base class adds one too. Please add a unit-test to ensure this is not happening.

Reply:
Added `ExpressionCompilerTests.EmitNullableAttribute_ExpressionType` test.